x-pack/plugins/ml/public/application/datavisualizer/index_based/components/field_data_row/index.ts

```diff
@@ -4,4 +4,4 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
-export { FieldsStats } from './fields_stats';
+export {};
```

Comment:
was this left in by accident?

Reply:
Removed here https://github.com/elastic/kibana/pull/87598/commits/8b99e8dd9f2085066c415ff58863d4aaec4b6f37